packages/nx-plugin/src/executors/internal/cli.ts

```diff
@@ -12,15 +13,49 @@ export function createCliCommandString(options?: {
   )}`;
 }
 
+export function formatCommandLog({
+  command,
+  args = [],
+  env,
+}: {
+  command: string;
+  args: string[];
+  env?: Record<string, string>;
+}): string {
+  const logElements: string[] = [];
+  if (env) {
+    const envVars = Object.entries(env).map(
+      ([key, value]) =>
+        `${chalk.green(key)}="${chalk.blueBright(value.replaceAll('"', ''))}"`,
+    );
+    // eslint-disable-next-line functional/immutable-data
+    logElements.push(...envVars);
+  }
+  // eslint-disable-next-line functional/immutable-data
+  logElements.push(chalk.cyan(command));
+  if (args.length > 0) {
+    // eslint-disable-next-line functional/immutable-data
+    logElements.push(chalk.dim.gray(args.join(' ')));
+  }
+  return logElements.join(' ');
+}
```

Comment:
Following up on previous comment, there's a `formatCommandLog` in `utils` that could be copied over, so we don't diverge too much. I'd also prefer to stick to an immutable coding style - i.e. conditional array spread instead of mutating the array.

However, I see your version has an extra feature - prefixing environment variables :+1: Do you think you add that to the upstream implementation, then copy it over here? :slightly_smiling_face:

I'd prefer to maintain one robust implementation (albeit copy-pasted), rather than two diverging implementations which are incomplete in different ways.